inspirehep/utils/url.py

```diff
@@ -58,7 +58,10 @@ def is_pdf_link(url):
     except requests.exceptions.RequestException:
         return False
 
-    magic_number = next(response.iter_lines(1))
+    try:
+        magic_number = next(response.iter_lines(1))
+    except StopIteration:
+        return False
```

Comment:
I think there is a better way to do this... I remember discussing this with @kaplun. Let me search for it.